src/effects/Limiter.cpp

```diff
@@ -0,0 +1,93 @@
+/*  SPDX-License-Identifier: GPL-2.0-or-later */
+/*!********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  Limiter.cpp
+
+  Matthieu Hodgkinson
+
+**********************************************************************/
+
+#include "Limiter.h"
+#include "CompressorInstance.h" // A limiter is just a compressor with hard-coded parameters.
+#include "LimiterEditor.h"
+#include "LoadEffects.h"
+#include "ShuttleAutomation.h"
+
+const ComponentInterfaceSymbol EffectLimiter::Symbol { XO(
+   "Real-time Limiter") };
+
+namespace
+{
+BuiltinEffectsModule::Registration<EffectLimiter> reg;
+} // namespace
+
+const EffectParameterMethods& EffectLimiter::Parameters() const
+{
+   static CapturedParameters<
+      EffectLimiter, LimiterEditor::thresholdDb, LimiterEditor::kneeDb,
+      LimiterEditor::lookaheadMs, LimiterEditor::releaseMs,
+      LimiterEditor::makeUpDb>
+      parameters;
+   return parameters;
+}
+
+std::shared_ptr<EffectInstance> EffectLimiter::MakeInstance() const
+{
+   return std::make_shared<CompressorInstance>(*this);
+}
+
+bool EffectLimiter::CheckWhetherSkipEffect(const EffectSettings& settings) const
+{
+   // TODO
+   return Effect::CheckWhetherSkipEffect(settings);
+}
+
+EffectLimiter::EffectLimiter()
+{
+   SetLinearEffectFlag(true);
+}
+
+EffectLimiter::~EffectLimiter()
+{
+}
+
+ComponentInterfaceSymbol EffectLimiter::GetSymbol() const
+{
+   return Symbol;
+}
+
+TranslatableString EffectLimiter::GetDescription() const
+{
+   return XO("A limiter");
```

Comment:
@LWinterberg WDYT?

Reply:
Let's make sure we discuss the string I pushed in a follow-up commit.